static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -90,6 +96,9 @@ define([
                         new StickyMpu($adSlot, {top: 58}).create();
                     }
                 }
+                if (isAdfreeSurvey('variant') || isAdfreeSurvey('simple')) {
```

Comment:
It looks like we could maybe just test matches for both strings inside `isAdfreeSurvey`?